packages/db/src/migrations/20240927053214_leetcode-tagged-link.ts

```diff
@@ -0,0 +1,15 @@
+import { type Kysely } from 'kysely';
+
+export async function up(db: Kysely<any>) {
+  await db.schema
+    .alterTable('companies')
+    .addColumn('leetcode_tagged', 'varchar')
```

Comment:
let's change `leetcode_tagged` to `leetcode_tagged_slug` to follow the pattern that's used with `levels_fyi_slug`